components/LinkCustom/LinkCustom.tsx

```diff
@@ -0,0 +1,36 @@
+// Import necessary modules from Next.js and React
+import Link from 'next/link';
+import React from 'react';
+
+interface LinkCustomProps {
+  href: string;
+  children: any;
+  className?: string;
+  newTab?: boolean;
+}
+
+const LinkCustom = ({
+  href,
+  children,
+  className,
+  newTab = false,
+}: LinkCustomProps) => {
+  // Determine if the target attribute should be added based on the newTab prop
+  const targetAttr = newTab ? '_blank' : undefined;
```

Comment:
W: Instead of making this a separate variable, since you're only using `targetAttr` once here, could you not instead just directly place this ternary conditional inside `<Link/>`?

Reply:
I'm sure I can, I just thought it cleaner and easier to understand looking at it from the outside.

Reply:
I'm opening this back up; why should I change it? I think the clarity has greater value than removing one line of code.

Reply:
oh sorry I thought the thumbs up there let you know I understood your reasoning so I closed this conversation.  I agree with your reasoning here and you no longer need to change it :)

Reply:
AHA; okay cool beans haha. Thank you!